integration-tests/tests/preview-drs-uri.js

```diff
@@ -54,8 +54,6 @@ const testPreviewDrsUriFn = _.flow(
   await click(page, clickable({ textContains: testEntity.entityType }));
   console.log('opening preview for entity...');
   await click(page, elementInDataTableRow(testEntity.name, testEntity.attributes.file_uri));
-  console.log('waiting for no spinners in preview modal...');
-  await waitForNoSpinners(page);
   console.log('verifying text in preview modal...');
   await findText(page, 'Filename');
   await findText(page, 'File size');
```

Comment:
the `waitForNoSpinners` seems extraneous. Line 56 clicks on the link to open the modal. The modal has a spinner while it resolves the DRS URI; once resolved, it renders "Filename: …" and "File size: …"

So, we could:
* wait for the spinner to disappear
* wait for the expected text, which will be present as soon as the spinner disappears

or just:
* wait for the expected text

Reply:
It'd be nice if this works; that logic makes sense to me although I do see that `waitForNoSpinners()` was being used in a similar context in other PRs relatively recently so I wonder if there's more going on here. Maybe the timeout on `findText()`'s `waitFor()` is shorter than that of `waitForNoSpinners()`?

I'd be interested to hear what @msilva-broad thinks about this.

Reply:
I'd also be fine putting this `waitForNoSpinners` back, if anyone feels it's important. I originally thought it was this spinner failing the test, so I removed it … but on closer inspection I'm pretty sure it was a spinner during cleanup.